library/composer

```diff
@@ -3,12 +3,12 @@
 Maintainers: Composer (@composer), Rob Bast (@alcohol)
 GitRepo: https://github.com/composer/docker.git
 
-Tags: 1.8.4, 1.8, 1, latest
-Architectures: amd64, arm32v6, arm64v8, i386, ppc64le
-GitCommit: ebbb9efd87b78cf9b837f97105bcb6c5f0ee44ef
+Tags: 1.8.5, 1.8, 1, latest
+Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le
+GitCommit: be36910e4d2a3b5a055d2075982e04c835c5db2e
```

Comment:
This commit doesn't appear to be part of any branches on the repo -- is it the right one?

Reply:
Let me investigate what went wrong here.

Reply:
Sorry, I merged my branch on Github (using rebase) but forget to checkout master on my local machine. It was a late night :sweat: